client/blocks/edit-gravatar/style.scss

```diff
@@ -0,0 +1,23 @@
+.edit-gravatar__image-container {
+	display: inline-block;
+	position: relative;
+	line-height: 0;
+
+	&.is-uploading::after {
+		content: '';
+		position: absolute;
+			top: 0;
+			right: 0;
+			bottom: 0;
+			left: 0;
+		background-color: rgba( $white, 0.75 );
+	}
+}
+
+.edit-gravatar__spinner {
+	position: absolute;
+		top: 50%;
+		left: 50%;
+	transform: translate(-50%, -50%);
+	z-index: z-index( 'root', '.edit-gravatar__spinner' );
```

Comment:
Good point. I used z-index to make sure that the spinner was showing above the uploading overlay, which was using `after` to work. But, I can use `before` instead and not rely on z-index anymore, which I did in 8345a01. From what I can see, there doesn't cause a regression but I'll watch out for that.